SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/OptionTests.swift

```diff
@@ -211,5 +211,10 @@ class OptionTests: XCTestCase {
         XCTAssertEqual(reflectedSome!.field, 123)
         XCTAssertNil(reflectedNone)
     }
+    
+    func testFailableInitializer() {
```

Comment:
From now on all tests should say what we're verifying.

`/// Verify that ...`

This helps:
- new maintainers since it's easier to read English than code.
- forces the author to be clear about what behavior they're trying to test. Needing to explain ourselves makes us think critically about what we are doing and if we're covering enough cases.
- the reviewer noticing missing tests. For example, if the reviewer sees that what the test says that it is testing doesn't seem to match the code inside, they can comment on that. This tends to increase the quality of a test suite.

For simple stuff like this a sentence is fine.